DisplaySettings/DisplaySettings.json

```diff
@@ -1869,8 +2045,8 @@
                 "$ref": "#/definitions/result"
             }
         },
-        "setSurroundVirtualizer":{
-            "summary": "(Version 2) Sets the Surround Virtualizer boost. The Speaker/Surround Virtualizer enables a surround sound signal (including one generated by the Surround Decoder) to be rendered over a device with built-in speakers or headphones.",
+        "setSurroundVirtualizer(v2)":{
```

Comment:
Remove `(v2)` from the method signature. Application calls that include `(v2)` in the method signature will fail.

Reply:
As discussed keeping only version2 methods and removed (v2) from method.